plugins/wallabag.koplugin/main.lua

```diff
@@ -315,13 +334,62 @@ function Wallabag:getBearerToken()
     end
 end
 
+--- Get a JSON formatted list of articles from the server.
+-- The list should have self.article_per_sync item, or less if an error occured.
+-- If filter_tag is set, only articles containing this tag are queried.
+-- If ignore_tags is defined, articles containing either of the tags are skipped.
 function Wallabag:getArticleList()
     local filtering = ""
+    local ignoring = {}
     if self.filter_tag ~= "" then
         filtering = "&tags=" .. self.filter_tag
     end
-    local articles_url = "/api/entries.json?archive=0&perPage=" .. self.articles_per_sync .. filtering
-    return self:callAPI("GET", articles_url, nil, "", "")
+    if self.ignore_tags ~= "" then
+        local tag
+        -- take a not of all the tags that we should ignore
```

Comment:
The typo jumped out at me, but it can be a bit shorter/simpler too.
```suggestion
        -- Store all tags to ignore.
```